### PR TITLE
Copy Neptune Key from local

### PR DESCRIPTION
**Pull request by maciek-pioro**
# Description
todo

# Checklist
   - [x] Is this PR focused on a single feature without the possibility of being divided into smaller PRs?
   - [x] Are all variables set with logical defaults that are backward compatible?
   - [x] I attached link to neptune if it was necessary
   - [ ] Have you successfully executed the linter and tests?
   - [x] Are all functions concise and don't require splitting?
   - [x] Is there documentation for arguments and complex logic?
   - [x] Are there no temporary solutions in the code? Should a task be added to Trello to refine the code?
   - [x] Are all functions placed in appropriate files?
   - [x] Is the PR name meaningful and descriptive?
   - [x] Is PR description provided, or unnecessary?
  
Other questions:
   - [ ] If I made changes to the requirements or anything related to the Singularity image I did generate new image and upload a new image to the clusters
   - [ ] If this change is significant enough I notified all individuals working with this repository (@channel in slack channel)
   - [ ] If there is a need for a second reviewer I requested additional review 

# Reviewer's checklist:
   - [ ] Does the Neptune link work and does it actually compare the situation before and after the PR?
   - [ ] Do I understand the code?
   - [ ] Do I think another reviewer is not needed or I commented that this PR need another reviewer?

### CODE REVIEW

**Comment by maciek-pioro** [comment]
Two neptune experiments run from my PC, one as me, one as Kuba:
https://app.neptune.ai/o/pmtest/org/llm-random/runs/details?viewId=9a15aa36-1063-49c4-807d-53393abb118a&detailsTab=metadata&shortId=LLMRANDOM-2478&type=run
https://app.neptune.ai/o/pmtest/org/llm-random/runs/details?viewId=9a15aa36-1063-49c4-807d-53393abb118a&detailsTab=metadata&shortId=LLMRANDOM-2481&type=run

**Review by crewtool — APPROVED** [review]
LGTM

File: lizrd/scripts/grid.py [hunk 1]
```diff
@@ -51,7 +53,7 @@
 
     assert (
         len(set(interactive_options_per_config)) == 1
-    ), f"`interactive_debug` must be the same for all configs"
+    ), "`interactive_debug` must be the same for all configs"
```
Comment (Simontwice) [inline lizrd/scripts/grid.py:56]: how strict are we on the "one pr, one purpose" policy

File: lizrd/scripts/grid.py [hunk 2]
```diff
@@ -113,6 +115,11 @@
         job_name = training_args["name"]
         training_args["n_gpus"] = setup_args["n_gpus"]
 
+        singularity_env_arguments = make_singularity_env_arguments(
+            hf_datasets_cache_path=setup_args["hf_datasets_cache"],
+            neptune_key=args.neptune_key,
+        )
+
```
Comment (Simontwice) [inline lizrd/scripts/grid.py:122]: one either dies a "i write long, transprarent files" hero
or lives long enough to become a JS-like villian

**Pull request merged**